test/bin/test_fixpoint/code.md

```diff
@@ -31,3 +31,10 @@ module type Foo = sig type t end
 # Pipe.f ()
 - : unit
 ```
+
+```ocaml version>=4.08
```

Comment:
Could you move that to its own file, something like `ocaml-408-syntax.md`? I'd like to have one test case per file to ease the maintenance of the end to end tests!

I feel like such "hidden" test cases are harder to find and there's a chance one would duplicate them by accident!

Reply:
done